recipes/drms/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "drms" %}
+{% set version = "0.5.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "05147964834b1f24730c824addc6c2160dba0953beda26234113740ace3378fa" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 1
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.9.0
+    - pandas >=0.14.1
+    - six >=1.8.0
+  run:
+    - python
+    - numpy >=1.9.0
+    - pandas >=0.14.1
+    - six >=1.8.0
+
+test:
+  imports:
+    - drms
+
+about:
+  home: https://github.com/kbg/drms
+  license: MIT
+  license_family: MIT
+  license_file: ''
+  summary: Access HMI, AIA and MDI data with Python
+  description: |
+    The ``drms`` module provides an easy-to-use interface for accessing HMI, AIA and MDI data with Python.
+    It uses the publicly accessible `JSOC <http://jsoc.stanford.edu/>`_ DRMS server by default, but can also\nbe used with local `NetDRMS <http://jsoc.stanford.edu/netdrms/>`_ sites.
+    More information, including a detailed tutorial is available on `Read the Docs <http://drms.readthedocs.io/>`_.
+  doc_url: http://drms.readthedocs.io/
+  dev_url: https://github.com/kgb/drms
+
+extra:
+  recipe-maintainers:
+    - kbg
```

Comment:
Is `kbg` willing to maintain? If not, please remove the name

Reply:
@SolarDrew can you add me and @nabobalis while you are at it? thanks.

Reply:
I've not heard anything from him, so I'll remove it. No problem @Cadair, I'll do that.

Reply:
Sorry @SolarDrew, I must have forgotten to write a reply. I'm ok with (co-)maintaining the recipe, so feel free to keep me as a maintainer.

Reply:
No problem, I'll put you back on the list :)